### PR TITLE
fix(dependencies): adds babel-runtime as dependency

### PR DESCRIPTION
**Pull request by goldcaddy77**
Missing `babel-runtime` dependency.

screen shot 2017-03-27 at 5 57 50 pm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,15 +23,16 @@
     "prepublish": "npm run build"
   },
   "dependencies": {
+    "babel-runtime": "^6.23.0",
     "chalk": "^1.1.3",
     "cli-table2": "^0.2.0",
     "commander": "^2.9.0",
+    "fs-promise": "^1.0.0",
```
Comment (goldcaddy77) [inline package.json]: Reordering below done automatically by npm.

Comment (miguelmich) [inline package.json]: @goldcaddy77 thanks for this fix :)

**Comment by miguelmich** [comment]
@goldcaddy77 when do you see this error?

**Comment by goldcaddy77** [comment]
Just calling the executable by itself will cause the issue.  I'm guessing when you install in DEV mode you're getting this automatically

```
Install with npm: npm install -g jira-cl
jira  <-- fails here
```

**Comment by miguelmich** [comment]
That's interesting I a fresh Digital Ocean droplet it install babel runtime only installing the package like you did, maybe it's because of your nodejs or npm version.

Thanks again :)

**Pull request merged**